CHANGES.rst

```diff
@@ -18,10 +20,10 @@ dev (master)
 
 * ``pip install urllib3[secure]`` will install Certifi and
   PyOpenSSL as dependencies. (Issue #678)
-  
+
```

Comment:
I will never understand why text editors don't have "trim trailing whitespace" as on by default. Is the default assumption really that we have to cater to people writing in Whitespace? Or can we assume that a developer who wants to write Whitespace can find the option and switch it, thereby saving us from novice devs accidentally committing trailing whitespace so that they appear in my diffs later?

I had a point here but I don't remember what it was.